src/astro/databases/postgres.py

```diff
@@ -110,3 +126,62 @@ def table_exists(self, table: Table) -> bool:
 
         inspector = sqlalchemy.inspect(self.sqlalchemy_engine)
         return bool(inspector.dialect.has_table(self.connection, table.name, schema))
+
+    def combine_tables(
+        self,
+        source_table: Table,
+        target_table: Table,
+        source_to_target_columns_map: Dict[str, str],
```

Comment:
Could this parameter be optional?
A widespread use case here would be to combine tables with the same columns.

Reply:
@kaxil @mikeshwe any thoughts?